### PR TITLE
moving code to tools

### PR DESCRIPTION
**Pull request by lkuligin**

### CODE REVIEW

File: libs/tools/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 LangChain, Inc.
```
Comment (efriis) [inline libs/tools/LICENSE:3]:
```suggestion
Copyright (c) 2024 LangChain, Inc.
```

File: libs/tools/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+[tool.poetry]
+name = "langchain-google-tools"
+version = "0.0.1"
```
Comment (efriis) [inline libs/tools/pyproject.toml:3]:
```suggestion
version = "0.1.0"
```

File: libs/tools/pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain-google"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain-google/tree/main/libs/tools"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4.0"
```
Comment (efriis) [inline libs/tools/pyproject.toml:14]: does it not support 3.8?

**Review by efriis — APPROVED** [review]
Some small nits otherwise lgtm!

File: libs/tools/pyproject.toml (hunk 3 above)
Comment (lkuligin) [reply libs/tools/pyproject.toml]: fixed

**Pull request merged**